### ISSUE TITLE
control+upArrow and control+downArrow commands don't appear in the User Guide

### ISSUE THREAD

**Issue by danidiaz** [nvaccess/nvda#14192]
### Steps to reproduce:

- Open NVDA.
- Open a browser and navigate to a page that has multiple paragraphs.
- Enter control+upArrow when there's a previous paragraph, or control+downArrow when there's a next paragraph.

### Actual behavior:

control+upArrow moves the caret to the previous paragraph and reads it.

control+downArrow moves the caret to the next paragraph and reads it.

### Expected behavior:

These commands should appear in the User Guide. They are mentioned in [other guides](https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts).

#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

NVDA 2022.2.3

#### Windows version:

Windows 11.

**Comment by CyrilleB79** [comment]
Hi @danidiaz

The current usage in NVDA documentation is to only document shortcuts specific to NVDA, not Windows shortcuts.

For now (as long as #13798 is not merged), `control+upArrow` and `control+downArrow` are strictly windows shortcuts, not shortcuts added by NVDA. Actually, you cannot modify these shortcuts in Input Gesture dialog.

Thus I would recommend to close this issue without modifying anything in the documentation.

In addition, NVAccess sells training material to use Windows with NVDA, which, I think, covers this type of topic.

Cc @seanbudd, @Qchristensen for confirmation and to close this ticket if needed.

Note: The alternative guide linked in the initial description of this issue is not totally accurate, e.g. it does not distinguish commands related to caret and to review cursor..

**Comment by seanbudd** [comment]
Agreed, closing. This was yet to be triaged.

**Comment by Qchristensen** [comment]
And just for reference, you can find control+up arrow / control+down arrow discussed in the context of text navigation in Basic Training for NVDA, Topic 6.2 More Navigation Commands.

It is also in the Microsoft Word with NVDA module in Topic 2.3 Writing Text
